taskcluster/ci/docker-image/kind.yml

```diff
@@ -0,0 +1,14 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - taskgraph.transforms.docker_image:transforms
+    - taskgraph.transforms.cached_tasks:transforms
+    - taskgraph.transforms.task:transforms
+
+jobs:
+    screenshots: {}
```

Comment:
This empty dict just means we're building the screenshots docker image with the default values. That is to say: it'll find `taskcluster/docker/screenshots/Dockerfile` and will know what to do with it.